client/src/task.rs

```diff
@@ -1,3 +1,4 @@
+use prio::vdaf::prio3::optimal_chunk_length;
```

Comment:
The changes in this file duplicate some logic from `src/entity/task/vdaf.rs` (on the `divviup-api` server side) fro computing histogram lengths and faulting in the chunk size. That's unfortunate, but doing better would require introducing a common representation of VDAFs across `divviup-cli`, `divviup-client` and the `divviup-api` backend. That might be a good idea in any case, but I don't want to do it in this PR.